Tools/autotest/common.py

```diff
@@ -5938,7 +5940,11 @@ def check_logs(self, name):
             shutil.move(log, newname)
             save_binaries = True
         if save_binaries:
-            util.run_cmd('/bin/cp build/sitl/bin/* %s' % to_dir,
+            if self.sitl_32bit:
+                sitl_dir = "sitl_32bit"
+            else:
+                sitl_dir = "sitl"
+            util.run_cmd('/bin/cp build/' + sitl_dir + '/bin/* %s' % to_dir,
```

Comment:
```suggestion
            util.run_cmd('/bin/cp build/%s/bin/* %s' % (' + sitl_dir + ', to_dir),
```

Reply:
This should actually be a call from a python library to copy the thing around.

Reply:
no longer exists